company/communication.md

```diff
@@ -60,11 +60,11 @@ Pro-tips:
 
 ### Current meetings
 
-#### Tuesday Kick Off
+#### Wednesday Team meeting
 
-Weekly kick off meetings have the intention to set goals for the week, create
-accountability, and discover blockers before they occur. Each participant will
-get the opportunity to share:
+Weekly, a meeting is held to set goals for the week, create accountability, and
+discover blockers before they occur. Each participant will get the opportunity
+to share:
```

Comment:
In previous discussions around this, I believe the suggestion was for the meeting to be less on the critical path of the way we work; that its main focus is an opportunity for the team to see each other and speak about what's on their mind/things they want to share.

I do not think the goal-setting aspect is part of it - that should be part of the standup responses on an individuals first working day of the week.